src/main/java/seedu/address/model/recipe/Recipe.java

```diff
@@ -94,11 +94,21 @@ public boolean isSameRecipe(Recipe otherRecipe) {
                 && otherRecipe.getName().equals(getName())
                 && otherRecipe.getInstruction().equals(getInstruction())
                 && otherRecipe.getRecipeImage().equals(getRecipeImage())
-                && otherRecipe.getIngredient().equals(getIngredient())
+                && isSameIngredients(otherRecipe.getIngredient())
                 && otherRecipe.getCalories().equals(getCalories())
                 && otherRecipe.getTags().equals(getTags());
     }
 
+    private boolean isSameIngredients(ArrayList<Ingredient> otherIngredients) {
```

Comment:
does this check for different ordering of ingredients?

Reply:
no should I check for diff ordering?